apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html

```diff
@@ -8,36 +8,46 @@ <h3>Export Daybook</h3>
         <form name="addShareEmailForm" novalidate class="" autocomplete="off">
             <div class="pdB2 clearfix">
                 <label class="w100">Type: </label>
-                <label class="radio-inline pd0 w100">
-                    <input type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected" [value]="emailTypeMini"
-                        class="radio_theme cp">Mini
-                </label>
-                <label class="radio-inline pd0 mrL2">
-                    <input type="radio" name="emailTypeSelected" [(ngModel)]="emailTypeSelected"
-                        [value]="emailTypeDetail" class="radio_theme cp">Detailed
-                </label>
+
+                <span class="pd0 mrR3">
+                    <input id="Mini" class="radio-custom cp ml0" name="emailTypeSelected" type="radio" [(ngModel)]="emailTypeSelected" [value]="emailTypeMini">
```

Comment:
Keep IDs in camelCase

Reply:
@prateek-giddh  Mini id name changed to mini